static/templates/draft_table_body.hbs

```diff
@@ -17,9 +17,26 @@
                     {{t 'No drafts.'}}
                 </div>
 
-                {{#each drafts}}
-                    {{> draft}}
-                {{/each}}
+                <div id="split-drafts">
+                    <div id="drafts-from-conversation">
+                        <h2>{{narrow_drafts_header}}</h2>
+                        {{#each narrow_drafts}}
+                            {{> draft}}
+                        {{/each}}
+                    </div>
+                    <div id="other-drafts">
+                        <h2>Other drafts</h2>
+                        {{#each other_drafts}}
+                            {{> draft}}
+                        {{/each}}
+                    </div>
+                </div>
+                <div id="all-drafts">
+                    {{!-- This section is only rendered when there are no narrow-specific drafts. --}}
+                    {{#each other_drafts}}
+                        {{> draft}}
+                    {{/each}}
+                </div>
```

Comment:
I think it'd probably be cleaner to just have the headings be hidden in that case via CSS, rather than having 2 copies of each draft in the DOM.

Reply:
Ohhh like, if there are no narrowed drafts, then hide both headings and show only the "other drafts"? Dang that means I didn't have to do any of that work to fix the tests to handle visible/not-visible :p I'll give that a try!